build_scripts/windows/scripts/build.cmd

```diff
@@ -114,18 +129,32 @@ for %%i in (%TEMP_SCRATCH_FOLDER%\*.whl) do (
     set ALL_MODULES=!ALL_MODULES! %%i
 )
 echo All modules: %ALL_MODULES%
-%BUILDING_DIR%\python.exe -m pip install --no-cache-dir %ALL_MODULES%
-%BUILDING_DIR%\python.exe -m pip install --force-reinstall --upgrade azure-nspkg azure-mgmt-nspkg
+%BUILDING_DIR%\python.exe -m pip install --no-warn-script-location --no-cache-dir %ALL_MODULES%
+%BUILDING_DIR%\python.exe -m pip install --no-warn-script-location --force-reinstall --upgrade azure-nspkg azure-mgmt-nspkg
 
 echo Creating the wbin (Windows binaries) folder that will be added to the path...
 mkdir %BUILDING_DIR%\wbin
 copy %REPO_ROOT%\build_scripts\windows\scripts\az.cmd %BUILDING_DIR%\wbin\
 if %errorlevel% neq 0 goto ERROR
 copy %REPO_ROOT%\build_scripts\windows\resources\CLI_LICENSE.rtf %BUILDING_DIR%
 copy %REPO_ROOT%\build_scripts\windows\resources\ThirdPartyNotices.txt %BUILDING_DIR%
-del %BUILDING_DIR%\Scripts\pip.exe
-del %BUILDING_DIR%\Scripts\pip3.exe
-del %BUILDING_DIR%\Scripts\pip3.6.exe
+
+: Delete some files we don't need
+rmdir /s /q %BUILDING_DIR%\Scripts
+for /f %%a in ('dir %BUILDING_DIR%\Lib\site-packages\*.egg-info /b /s /a:d') do (
+    rmdir /s /q %%a
+)
+for /d /r %BUILDING_DIR%\Lib\site-packages %%d in (__pycache__) do (
+    if exist %%d rmdir /s /q "%%d"
+)
+:: Use universal files and remove Py3 only files
+pushd %BUILDING_DIR%\Lib\site-packages\azure\mgmt
```

Comment:
Why "mgmt" only? There is already some py3 in dataplane generated (eventgrid, keyvault, etc.)